src/helm/proxy/clients/client.py

```diff
@@ -146,3 +147,13 @@ def cleanup_tokens(tokens: List[str], tokenizer_name: Optional[str] = None) -> L
     Applies `cleanup_str` to each token in `tokens`.
     """
     return [cleanup_str(token, tokenizer_name) for token in tokens]
+
+
+def generate_uid_for_multimodal_prompt(prompt: MultimediaObject) -> str:
+    """Generates a unique identifier for a given multimodal prompt."""
+    return "".join(
+        [
+            media_object.text if media_object.is_type(TEXT_TYPE) and media_object.text else str(media_object.location)
```

Comment:
No action needed: This assumes that the file is static and does not get modified, which should be true in most cases... otherwise we'd need a last modified time and / or checksum.

Reply:
Yes, I think it's true for most cases. If a user needs to modify `MediaObject`, then I think it would be a `file/MediaObject`.